DSCResources/VE_XD7SiteLicense/VE_XD7SiteLicense.psm1

```diff
@@ -62,12 +62,12 @@ function Get-TargetResource {
                 LicenseEdition = $xdSiteConfig.ProductEdition;
                 LicenseModel = $xdSiteConfig.LicensingModel;
                 TrustLicenseServerCertificate = !([System.String]::IsNullOrEmpty($xdSiteConfig.MetaDataMap.CertificateHash));
-                Ensure = $using:Ensure;
+                Ensure = $Ensure;
             };
 
             return $targetResource;
         } #end scriptblock
-
+        <# 
```

Comment:
Remove comment and implement the `& $invokeCommandParams.ScriptBlock` call as per all the other functions.